Adyen/__init__.py

```diff
@@ -45,6 +46,7 @@ def __init__(self, **kwargs):
         self.dataProtection = AdyenDataProtectionApi(client=self.client)
         self.transfers = AdyenTransfersApi(client=self.client)
         self.storedValue = AdyenStoredValueApi(client=self.client)
+        self.configuration = AdyenBalancePlatformApi(client=self.client)
```

Comment:
I think the word "configuration" is a bit too broad for this service. It could cause some confusion with the library's configuration or even platform config (e.g. Management API). How about:

```suggestion
        self.balancePlatform = AdyenBalancePlatformApi(client=self.client)
```